airflow/www/utils.py

```diff
@@ -228,7 +228,7 @@ def task_instance_link(attr):
     dag_id = attr.get('dag_id')
     task_id = attr.get('task_id')
     execution_date = attr.get('dag_run.execution_date') or attr.get('execution_date') or timezone.utcnow()
-    url = url_for('Airflow.task', dag_id=dag_id, task_id=task_id, execution_date=execution_date)
+    url = url_for('Airflow.task', dag_id=dag_id, task_id=task_id)
```

Comment:
```suggestion
    url = url_for('Airflow.task', dag_id=dag_id, task_id=task_id,  execution_date=execution_date.isoformat())
```
This suggestion was not applied, I had thought I applied it, surprised that I didn't. https://github.com/apache/airflow/pull/18418#discussion_r713520604